REFERENCE.md

```diff
@@ -475,3 +483,35 @@ See [clientloglimit](https://chrony.tuxfamily.org/doc/3.4/chrony.conf.html#clien
 
 Default value: `undef`
 
+##### `rtcsync`
+
+Data type: Boolean
+
+Periodically sync syste time to RTC
```

Comment:
typo in `system`

Reply:
Yes, good catch.  Do you prefer that I add a small typo-fix commit, or a rebase -i and a force-push?